### ISSUE TITLE
Group Comics by Series Arc

### ISSUE THREAD

**Issue by TimOliver** [iComics/iComics-Issues#83]
Not sure how to do this. Maybe as a metadata tag inside comic files, or even just specially marking folders on disk, but the ability to section off a comic series by their specific story arcs.

**Comment by chinagreenelvis** [comment]
Comic files don't have metadata. Currently, I have to split up collections into folders. A subfolder feature would be even better.

**Comment by TimOliver** [comment]
Nested folders would definitely let people set this up themselves. I'm planning on adding this when I do the overhaul. :)

This is something another user suggested. There are standards for comic book meta data out there, so it's worth exploring.

**Comment by chinagreenelvis** [comment]
https://wiki.mobileread.com/wiki/CBR_and_CBZ#Metadata

Interesting, I didn't know that!

**Comment by TimOliver** [comment]
Yeah! It's not really huge, but there's definitely at least a few people out there who meticulously curate their comic collection with metadata.

I'm still working out how to actually go about implementing it and to make it users could modify the metadata from the app. Editing ZIP files is easy, but RAR is a proprietary format that needs a license in order to be able to modify them. :/
